modules/ROOT/pages/dataweave-type-reuse-java-classes.adoc

```diff
@@ -0,0 +1,70 @@
+= Reusing Types from Java Classes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the import directive with the `java!` prefix when loading a Java class. The prefix `java!` tells DataWeave to use the Java module loader. When loaded, you can call constructors, variables, and functions, see xref:dataweave-cookbook-java-methods.adoc[]. The class type is also available. This is exposed with the type name `Class`.
```

Comment:
```suggestion
Use the import directive with the `java!` prefix when loading a Java class. The prefix `java!` tells DataWeave to use the Java module loader. After loading the module, you can call constructors, variables, and functions (see xref:dataweave-cookbook-java-methods.adoc[]). The class type is also available. This is exposed with the type name `Class`.
```

Reply:
Question about this:
"The class type is also available. This is exposed with the type name `Class`."

What does "This" in "This is exposed" refer to? The class type?

Reply:
Are we saying that the type of the Java class is exposed through the type name `Class`?

Reply:
generally, "this" should alway modify something

Reply:
Yes, we are!